lib/shared/robot/README.md

```diff
@@ -0,0 +1,123 @@
+The "robot" mechanism
+=====================
+
+As discussed in `lib/child-writer`, communication between the main Etcher
```

Comment:
I think it'd make sense to link to `lib/child-writer` here, enabling quick navigation (i.e. for people not familiar with the directory structure)